### DISCUSSION TITLE
Don't halt on "Undefined control sequence"

### DISCUSSION THREAD

**Discussion by Frollamma** [tectonic-typesetting/tectonic#971] [Q&A] [answered]
During compiling I often get the error "Undefined control sequence", to be precise the terminal shows:
```
error: main.tex:1647: Undefined control sequence
note: Writing `c:/Users/my_latex_project_path\main.log` (58.99 KiB)
error: halted on potentially-recoverable error as specified
```
I use LaTeX mostly for live note taking, I have to type fast, so typos happens a lot to me and I get this error for this reason. I don't want to stop my workflow for a typo, is there a way to show a warning in place of an error for "Undefined control sequence"? I know this is possible because that's exactly what Overleaf does.
Thanks for any help

**Comment by Neved4** [comment, marked as answer]
Hey @Frollamma thanks for the issue 🖤
Have you tried to use ` -Z continue-on-errors`?

**Reply by Frollamma** [reply]
@mnrvwl thank you for your answer! I just tried and it works from command line!
But it doesn't with my current LaTeX Workshop configuration (I'm using vscode) :
```
"latex-workshop.latex.recipes": [        
        {
            "name": "tectonic (skip errors)",
            "tools": [
                "tectonic -Z continue-on-errors"
            ]
        },
        {
            "name": "tectonic",
            "tools": [
                "tectonic"
            ]
        },
]
```
When using the first recipe I get this error `Skipping undefined tool "tectonic -Z continue-on-errors" in recipe "tectonic (skip errors)."`

**Reply by Frollamma** [reply]
If someone ever need it: I added
```
{
    "name": "tectonic-skip",
    "command": "tectonic",
    "args": [
        "-Z",
        "continue-on-errors",
        "--synctex",
        "--keep-logs",
        "%DOC%.tex"
    ],
    "env": {}
}
```
to `latex-workshop.latex.tools`, so that I created a tool named `tectonic-skip`, then I added
```
{
    "name": "tectonic (skip errors)",
    "tools": [
        "tectonic-skip"
    ]
}
```
to `latex-workshop.latex.recipes`. And it worked as expected!
